plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-button.tsx

```diff
@@ -0,0 +1,92 @@
+/**
```

Comment:
This was based on [payment-methods.tsx](https://github.com/woocommerce/woocommerce/blob/6e8805fd1547157c63f89ff0e148eca064df476b/plugins/woocommerce/client/admin/client/settings-payments/components/payment-method.tsx), that file could probably be removed soon since it's no longer used

Reply:
Good idea to move this to a component!